src/index.html

```diff
@@ -11,7 +11,128 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post" >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="labels">
+          Surname:
+          <input type="text"
+            name="surname"
+            required
+            autocomplete="off"
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="surname"
            required
            autocomplete="off"
          >
```
not fixed, fix everywhere